  apf_navigation_controller:
    ros__parameters:
      max_linear_vel : 0.15       # 0.15
      kp : 2.0                    # 2.0
      eta : 2.0                   # 2.0
      repulsion_radius : 0.3      # 1.5
      goal : [3.5, 0.5]